go.work

```diff
@@ -1,4 +1,4 @@
-go 1.22
+go 1.23.0
```

Comment:
1.23.2 as we typically set for go.mod?

Reply:
minor version doesn't matter as long as there are no changes affecting build in minor go versions. so `.0` should work here as well for the lifecycle of 1.23